apps/bookmark/save.html

```diff
@@ -47,6 +47,7 @@ <h1 id="edit-bookmark-header" data-l10n-id="edit-bookmark-title"></h1>
       </header>
       <form id="bookmark-form">
         <img id="bookmark-icon">
+        <a id="bookmark-url"></a>
```

Comment:
We have a UX spec for this, see:

Interaction spec
https://mozilla.app.box.com/s/fhlmfa4xyooqpwaaaczc/1/2545278347/24886256343/1

Visual spec
https://mozilla.app.box.com/s/fnuq2vthajkvodwemrq1

It says the URL should be visible, but not clickable or editable.

Thanks for taking this on, hope this helps.